packages/experimental/lazy/lazy.js

```diff
@@ -11,18 +11,14 @@ var web = require('./../../core/core');
  * Web container for all components and services.
  */
 var lazy = (function lazy() {
-    var lazy = {
-        componentRegistry: {},
-        serviceRegistry: {},
-        deferredComponents: {},
-        deferredServices: {},
-    };
+    var lazy = {};
```

Comment:
```suggestion
    var lazy = {
        componentRegistry: {},
        serviceRegistry: {},
        deferredComponents: {},
        deferredServices: {},
    };
```

This object should stay as before.